client/my-sites/checkout/composite-checkout/hooks/use-create-payment-methods/index.ts

```diff
@@ -435,9 +435,10 @@ export default function useCreatePaymentMethods( {
 		storedCards,
 	} );
 
+	// The order mirrors the order of Payment Methods in Checkout.
```

Comment:
I think it's actually safe to say that this order _is_ the order of Payment Methods in Checkout. It doesn't mirror that order because the order doesn't exist anywhere else.